src/ui/server/controllers/insurance/your-buyer/outstanding-or-overdue-payments/validation/rules/amount-overdue.test.ts

```diff
@@ -0,0 +1,33 @@
+import amountOverdueRules, { MINIMUM } from './amount-overdue';
```

Comment:
Whilst we're moving things around, can we move this `MINIMUM` into `MINIMUM_CHARACTERS` constants please